README.md

```diff
@@ -24,36 +24,29 @@ Worker also expands the set of tables that quartz uses with additional tables to
 5. Run Piezo as specified in [Running](#running).
 
 ###Building
-You must have [sbt](http://www.scala-sbt.org/) version 0.13.0 or higher to build the worker project.
+You must have [sbt](http://www.scala-sbt.org/) version 0.13.0 or higher installed.
 
-To compile the project run
+From the [worker](worker) directory,
 
-`make compile`.
+`sbt compile` compiles sources.
 
-To package the project into a jar run
-
-`make package`.
-
-To collect all dependencies into a single folder (target/staged) run
-
-`make stage`
+`sbt packageBin` creates a JAR.
 
 ###Configuration
 ####JVM properties
 * `org.quartz.properties` - [Quartz scheduler library config file](http://quartz-scheduler.org/documentation/quartz-2.2.x/configuration/)
-* `logback.configurationFile` - [Logback config file](http://logback.qos.ch/manual/configuration.html)
```

Comment:
Anyone who uses Dispatch as an anti-example has my vote. Although, you can
hardly call println logging. :)

On Mon, Aug 17, 2015 at 10:31 AM, Paul Draper <notifications@github.com>
wrote:

> In README.md
> <https://github.com/lucidsoftware/piezo/pull/32#discussion_r37206901>:
>
> >
> >  ###Configuration
> >  ####JVM properties
> >  * `org.quartz.properties` - [Quartz scheduler library config file](http://quartz-scheduler.org/documentation/quartz-2.2.x/configuration/)
> > -* `logback.configurationFile` - [Logback config file](http://logback.qos.ch/manual/configuration.html)
>
> The purpose of SLF4J is that libraries don't have to impose their choices
> of logging on end programs.
>
> Everyone logs against the very simple facade, and the end-program provides
> a SLF4J binding on the classpath ("configuration by classpath") with the
> actual implementation/configuration.
>
> E.g. a google-api-client logs against SLF4J, and the user can configure it
> with Logback, or Log4J, or java.util.logging, etc.
>
> Anti-example: Dispatch, which just kinda spews its logs wherever.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lucidsoftware/piezo/pull/32/files#r37206901>.
>